openlibrary/plugins/openlibrary/js/add-book.js

```diff
@@ -154,3 +160,19 @@ function autoCompleteIdName(){
         document.getElementById('id_name').value = '';
     }
 }
+
+function checkIsbn() {
+    const isbn = document.getElementById('id_value').value;
+    fetch(`/isbn/${isbn}.json?high_priority=false`)
+        .then((resp) => {
+            if (resp.status !== 200) {
+                throw new Error(`Failed to fetch ISBN. Status code: ${resp.status}`)
+            }
+            return resp.json()
+        })
+        .then((data) => {
+            const foundDiv = document.getElementById('id-found');
+            foundDiv.classList.remove('hidden');
+            foundDiv.innerHTML = `${data.title} - An existing book was found with ISBN ${isbn}: <a href="/isbn/${isbn}" style="color:red">openlibrary.org/isbn/${isbn}</a>`;
```

Comment:
```suggestion
```
Instead of all of this, you'll want to use `displayIsbnError` to display the message.